test/e2e/networks/ci.toml

```diff
@@ -39,67 +39,68 @@ mode = "seed"
 perturb = ["restart"]
 
 [node.validator01]
+perturb = ["disconnect"]
 seeds = ["seed01"]
 snapshot_interval = 5
-perturb = ["disconnect"]
 
 [node.validator02]
-seeds = ["seed01"]
 database = "goleveldb"
-privval_protocol = "tcp"
+mempool_version = "v2"
```

Comment:
why does this use v2 instead of v1 / priority?

Reply:
to test the cat mempool legacy setting

Reply:
Does a different network test use the v1 (a.k.a prioritized mempool)? Since that's the current default, it seems like it should also have e2e test coverage.

Reply:
the ci.toml uses a mixture of v0,v1 and v2. I opted to do the mixture to make sure they work together as mainnet could end up in a similar situation.

Reply:
Sounds good, ty